### PR TITLE
Make contract editor function calls, example blocks, and example blocks' function calls immovable & deletable.

### PR DESCRIPTION
**Pull request by bcjordan**
Also fixes render-on-modal-open for Firefox [#90225590].

### CODE REVIEW

File: blockly-core/core/ui/block.js [hunk 1]
```diff
@@ -1336,7 +1336,7 @@ Blockly.Block.prototype.setUserVisible = function(userVisible, opt_renderAfterVi
   }
   // Apply to all children recursively
   this.childBlocks_.forEach(function (child) {
-    child.setUserVisible(userVisible);
+    child.setUserVisible(userVisible, opt_renderAfterVisible);
```
Comment (bcjordan) [inline blockly-core/core/ui/block.js]: This was supposed to be passed down recursively. This successfully resolves the Firefox rendering issue [#90225590].

Before:

image


After:

image

File: blockly-core/core/ui/contract_editor/contract_editor.js [hunk 2]
```diff
@@ -310,10 +310,22 @@ Blockly.ContractEditor.prototype.moveExampleBlocksToModal_ = function (functionN
...
+    exampleCall.setDeletable(false);
     this.exampleBlocks.push(movedExampleBlock);
   }, this);
 };
 
+/**
+ * @override
+ */
+Blockly.ContractEditor.prototype.moveToModalBlockSpace_ = function (block) {
+  var newBlock = Blockly.ContractEditor.superClass_.moveToModalBlockSpace_.call(this, block);
```
Comment (Bjvanminnen) [inline blockly-core/core/ui/contract_editor/contract_editor.js]: something_.somethingelse_ makes it look like we're accessing things we shouldn't be. I would think there's a proper closure way to access super class?

Comment (Bjvanminnen) [inline blockly-core/core/ui/contract_editor/contract_editor.js]: goog.base()?

**Comment by Bjvanminnen** [comment]
one comment, otherwise lgtm

File: blockly-core/core/ui/contract_editor/contract_editor.js (hunk 2 above)
Comment (bcjordan) [inline blockly-core/core/ui/contract_editor/contract_editor.js]: Renamed to omit trailing _

**Pull request merged**